sfdx-project.json

```diff
@@ -4,8 +4,8 @@
             "path": "force-app",
             "default": true,
             "package": "Clayton Copado Integration",
```

Comment:
Could we call this integration "Clayton for Copado"?

Reply:
This would be consistent with how we are naming our other integrations

Reply:
Yes, I just don't know the impact, but I can modify that